awesomeCreateApp.js

```diff
@@ -47,4 +51,8 @@ const showEndProcessText = () => {
   Promise.resolve()
 }
 
+const checkPackageInstalled = packageName => {
```

Comment:
1 - This could be an one line function

2 - this could return error when the create-react-app is installed, so should be better use `create-react-app --version` 💃